.github/actions/sls-deployment/action.yml

```diff
@@ -0,0 +1,88 @@
+name: sls deploy action
+description: Reusable sls deploy
+
+inputs:
+  environment-name:
+    description: 'The environment name'
+    required: true
+  app:
+    description: 'The application - crn-server or gp2-server'
+    required: true
+  AWS_ACCESS_KEY_ID:
+    description: 'AWS Access Key ID'
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    description: 'AWS Secret Access Key'
+    required: true
+  LIGHTSTEP_TOKEN:
+    description: 'Lightstep Token'
+    required: true
+  SQUIDEX_CLIENT_SECRET:
+    description: 'Squidex Client Secret'
+    required: true
+  SQUIDEX_SHARED_SECRET:
+    description: 'Squidex Shared Secret'
+    required: true
+
+outputs:
+  crn-api-url:
+    description: 'CRN API Url'
+    value: ${{ steps.setup.outputs.crn-api-url }}
+  crn-app-url:
+    description: 'CRN APP Url'
+    value: ${{ steps.setup.outputs.crn-app-url }}
+  gp2-api-url:
+    description: 'GP2 API Url'
+    value: ${{ steps.setup.outputs.gp2-api-url }}
+  gp2-app-url:
+    description: 'GP2 APP Url'
+    value: ${{ steps.setup.outputs.gp2-app-url }}
+
```

Comment:
Why do we need outputs?

Reply:
for
<img width="447" alt="image" src="https://user-images.githubusercontent.com/9073485/170462094-088a5901-0997-4753-ac84-cae9f6122f68.png">

in reusable-deployment